mne/decoding/ems.py

```diff
@@ -1,26 +1,121 @@
 # Author: Denis Engemann <denis.engemann@gmail.com>
 #         Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#         Jean-Remi King <jeanremi.king@gmail.com>
 #
 # License: BSD (3-clause)
 
 import numpy as np
 
+from .mixin import TransformerMixin, EstimatorMixin
+from .base import _set_cv
 from ..utils import logger, verbose
 from ..fixes import Counter
 from ..parallel import parallel_func
 from .. import pick_types, pick_info
 
 
+class EMS(TransformerMixin, EstimatorMixin):
```

Comment:
can you add a repr ?

In [3]: ems
Out[3]: <mne.decoding.ems.EMS at 0x119fc5810>

so far.